C/Quick Sort.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include<stdlib.h>
+void swap(int *x,int *y)
+{
+ int temp=*x;
+ *x=*y;
+ *y=temp;
+}
+int partition(int A[],int l,int h)
+{
+ int pivot=A[l];
+ int i=l,j=h;
+ 
+ do
+ {
+ do{i++;}while(A[i]<=pivot);
+ do{j--;}while(A[j]>pivot);
+ 
+ if(i<j)swap(&A[i],&A[j]);
+ }while(i<j);
+ 
+ swap(&A[l],&A[j]);
+ return j;
+}
+void QuickSort(int A[],int l,int h)
+{
+ int j;
+ 
+ if(l<h)
+ {
+ j=partition(A,l,h);
+ QuickSort(A,l,j);
+ QuickSort(A,j+1,h);
+ }
+}
+int main()
+{
+ int A[]={11,13,7,12,16,9,24,5,10,3},n=10,i;
+ 
+ QuickSort(A,n);

```

Comment:
Too few arguments

Reply:
it is just for example. User may exceed it according to the need.

Reply:
The function ```QuickSort ``` takes 3 arguments. 
Please provide 3 arguments or change the function to resolve compilation error.
Function call at Line 40 is passing only 2 arguments.